curp/src/server.rs

```diff
@@ -79,6 +79,20 @@ where
 }
 
 impl<C: Command + 'static, CE: CommandExecutor<C> + 'static> Rpc<C, CE> {
+    /// New `Rpc`
+    #[inline]
+    pub fn new(is_leader: bool, term: u64, others: Vec<String>, executor: CE) -> Self {
```

Comment:
please use this `new` function in the following `run` function to reduce the duplication.

Reply:
done